test/integration/query-interface.test.js

```diff
@@ -63,6 +63,23 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
           expect(tableNames).to.deep.equal(['my_test_table']);
         });
     });
+
+    if (dialect !== 'sqlite' && dialect !== 'postgres') {
```

Comment:
Postgre supports `CREATE DATABASE`.
The odd one out is SQLite. For that one you can use `ATTACH DATABASE 'blah.db' as my_db` although the test is somewhat unnecessary because the `sqlite_master` view is limited to each database.

Reply:
Postgres doesn't allow creating a table in another database

```sql
CREATE TABLE my_test_db.my_test_table2 (id INT)
```

Reply:
Googling the issue shows that one needs a new connection to the other DB. Seems a bit overkill so not worthwhile testing for that. But at least you should write a comment why these dialects are being excluded.